### PR TITLE
Production

### PR DESCRIPTION
**Pull request by codeninja**

### CODE REVIEW

File: config-docker.yaml [hunk 1]
```diff
@@ -3,36 +3,21 @@
 cache: redis
 
 redis:
-    ip: redisdb
+    ip: cryptostore-redis.uq1sqt.0001.use1.cache.amazonaws.com
```
Comment (bmoscon) [inline config-docker.yaml:6]: this looks like your specific instance

File: config-docker.yaml [hunk 2]
```diff
@@ -3,36 +3,21 @@
...
 
-storage: [arctic]
-arctic: mongodb://mongo:27017
+parquet:
+    del_file: true
+    S3:
+        endpoint: null
+        key_id: null
+        secret: null
+        bucket: quantfolio-ai-data-collection
```
Comment (bmoscon) [inline config-docker.yaml:20]: your specific bucket, should probably not be here

File: docker-compose.yml [hunk 3]
```diff
@@ -1,33 +1,19 @@
...
   cryptostore:
     logging:
       driver: "json-file"
       options:
         max-size: "2000k"
         max-file: "10"
-    build:
-      context: .
-      dockerfile: Dockerfile
+    image: 966108416930.dkr.ecr.us-east-1.amazonaws.com/ftx-perps:latest
```
Comment (bmoscon) [inline docker-compose.yml:9]: same

File: cryptostore/data/parquet.py [hunk 4]
```diff
@@ -66,7 +67,8 @@ def write(self, exchange, data_type, pair, timestamp):
 
         if self._write:
             for func, bucket, prefix, kwargs in zip(self._write, self.bucket, self.prefix, self.kwargs):
-                path = f'{exchange}/{data_type}/{pair}/{int(timestamp)}.parquet'
+                date = datetime.utcfromtimestamp(timestamp).strftime('%Y-%m-%d')
```
Comment (bmoscon) [inline cryptostore/data/parquet.py:70]: i think this is fine, so long as the documentation is updated to reflect this

File: quantfolio.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
+## Info
```
Comment (bmoscon) [inline quantfolio.md:1]: I'm not sure this file is needed

File: task-definition.json [hunk 6]
```diff
@@ -0,0 +1,123 @@
+{
```
Comment (bmoscon) [inline task-definition.json:1]: probably need to clean this up, remove your specific variables, and have a readme about how to use this

**Comment by codeninja** [comment]
Apologies @bmoscon, this was submitted to the root rather than our fork. It would be most appreciated if you would nuke this PR entirely. 

Thanks.

**Pull request closed without merging**